tests/monitoring/monitoring.go

```diff
@@ -281,6 +290,69 @@ var _ = Describe("[Serial][sig-monitoring]Prometheus Alerts", func() {
 		})
 	})
 
+	Context("VM migration metrics", func() {
+		var nodes *k8sv1.NodeList
+
+		BeforeEach(func() {
```

Comment:
I suggest to add `checks.SkipIfMigrationIsNotPossible()` since its being widely used in the e2e suites where migration should be tested.

Reply:
added